syntax = "proto3";

message SmokeStatus {
    float status = 1;
}

service Smoke {
    rpc turnOn(SmokeStatus) returns (SmokeStatus) {}
    rpc turnOff(SmokeStatus) returns (SmokeStatus) {}
}